infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PodsVolumes.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.namespace.pvc;
+
+import io.fabric8.kubernetes.api.model.Volume;
+import io.fabric8.kubernetes.api.model.VolumeBuilder;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
+import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment.PodData;
+
+/**
+ * Helps to works with Pods Volumes, like reference them to another PVC.
+ *
+ * @author Sergii Leshchenko
+ */
+public class PodsVolumes {
+
+  /**
+   * Changes all pods volumes witch referenced the specified PVC to reference new PVC.
+   *
+   * @param pods pods to change
+   * @param currentPVCName current PVC name for filtering pods volumes
+   * @param newPVCName new PVC name that should be used
+   */
+  public void changePVCReferences(
```

Comment:
Since this class has no internal state, consider making it static.

Reply:
Previously it was static but it was impossible to mock behavior of this class in tests since we don't use `PowerMock`

Reply:
Feel free to reopen if you have an idea how to rework tests with static method

Reply:
Ah I see, makes sense.